cpplint_clitest.py

```diff
@@ -176,29 +178,22 @@ class NoRepoSignatureTests(TemporaryFolderClassSetup, unittest.TestCase):
     """runs in a temporary folder (under /tmp in linux) without any .git/.hg/.svn file"""
 
     def get_extra_command_args(self, cwd):
-        return (' --repository %s ' % self._root)
-
-    def testChromiumSample(self):
-        self.checkAllInFolder('./samples/chromium-sample', 1)
-
-    def testVlcSample(self):
-        self.checkAllInFolder('./samples/vlc-sample', 1)
-
-    def testSillySample(self):
-        self.checkAllInFolder('./samples/silly-sample', 5)
-
-    def testBoostSample(self):
-        self.checkAllInFolder('./samples/boost-sample', 4)
-
+        return f' --repository {self._root} '
+
+    def _test_name_func(fun, _, x):
+        del fun
+        return f'test{x.args[0].capitalize()}Sample-{x.args[1]}'
+
+    @parameterized.expand([(folder, case[:-4])
```

Comment:
Do we still need https://github.com/wolever/parameterized that has had no update in two years, when pytest has https://docs.pytest.org/en/stable/how-to/parametrize.html#parametrize builtin?

Reply:
That was the first thing I tried. I couldn't get that to work with the unittest-module–subclass setup in this repo. (Plas speling is very importent.)

Reply:
It is not tested on Python 3.12, 3.13, and pytest > v3 (current is v8).  See the open issues and
* wolever/parameterized#182

Reply:
Well, this seems like the best thing we have. We could always find an alternative solution later. It doesn't seem like parameterized's actual functionality we use has been impacted by their indeed-worrying neglect yet. (This was a reason I also tried the subtests approach, which works without additional modules but doesn't allow for -k filtering.)

Reply:
With #332 I do not see why we need `unittest.TestCase` classes.  I will do more work on that today.

Reply:
I think we'd have a less messy branch history if we just merge this now while you work onthat.

Reply:
#322 has now removed all instances of `unittest.TestCase`.